src/cellular/dataUpload.hpp

```diff
@@ -44,9 +44,35 @@ class DataUpload : public Task{
+     * |-------------------|----------------------------|----------|--------------------------------|-------------------------|--------------------|------------------------|
+     * | 0                 | 0                          | 0        | 0                              | 0                       | 0                  | 1                      |
+     * | 0                 | 0                          | 0        | 1                              | 0                       | 0                  | 1                      |
+     * | 0                 | 0                          | 1        | 0                              | 0                       | 0                  | 1                      |
+     * | 0                 | 0                          | 1        | 1                              | 0                       | 0                  | 1                      |
+     * | 0                 | 1                          | 0        | 0                              | 0                       | 0                  | 1                      |
+     * | 0                 | 1                          | 0        | 1                              | 0                       | 0                  | 1                      |
+     * | 0                 | 1                          | 1        | 0                              | 0                       | 0                  | 1                      |
+     * | 0                 | 1                          | 1        | 1                              | 0                       | 0                  | 1                      |
+     * | 1                 | 0                          | 0        | 0                              | 0                       | 0                  | 1                      |
+     * | 1                 | 0                          | 0        | 1                              | 0                       | 0                  | 1                      |
+     * | 1                 | 0                          | 1        | 0                              | 0                       | 0                  | 1                      |
+     * | 1                 | 0                          | 1        | 1                              | 0                       | 0                  | 1                      |
+     * | 1                 | 1                          | 0        | 0                              | 0                       | 0                  | 1                      |
+     * | 1                 | 1                          | 0        | 1                              | 1                       | 0                  | 0                      |
+     * | 1                 | 1                          | 1        | 0                              | 0                       | 1                  | 0                      |
+     * | 1                 | 1                          | 1        | 1                              | 0                       | 1                  | 0                      |
```

Comment:
How should I as a reader interpret this table?  Is this meant to be a truth table?  Truth tables cannot be used to output non-boolean outputs.